deployment/helm/node-feature-discovery/Chart.yaml

```diff
@@ -13,3 +13,7 @@ keywords:
   - node-labels
 type: application
 version: 0.2.1
+dependencies:
+  - name: crds
+    version: 0.0.0
```

Comment:
There's a drawback to use a version here: you need bump the version and `helm dependency update` with every release.

The versioning usually makes sense when adding charts from repos, with different lifecycles.

In our case, CRDs are always generated and copied under `crds` folder: https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/hack/update_codegen.sh#L44-L46

There's a 1-1 match in the lifecycle for this chart. Thus, the `version: 0.0.0` makes the trick to avoid extra steps.